src/Register.php

```diff
@@ -0,0 +1,45 @@
+<?php namespace Leean\Elements;
+
+use Leean\Elements\Options;
+
+/**
+ * Class to provide predefined ACF elements.
+ */
+class Register
+{
+	/**
+	 * The location for the elements.
+	 *
+	 * @var array
+	 */
+	private static $_location = [];
```

Comment:
No, its one location only.
But Im discussing with Adam if we need to remove it altogether or not.